lasagne/layers/conv.py

```diff
@@ -123,9 +123,9 @@ class Conv1DLayer(Layer):
         is equivalent to computing the convolution wherever the input and the
         filter overlap by at least one position.
 
-        ``'same'`` pads with half the filter size on both sides (one less on
-        the second side for an even filter size). When ``stride=1``, this
-        results in an output size equal to the input size.
+        ``'same'`` pads with half the filter size (rounded down) on both sides,
+        this results in an output size equal to the input size. Even filter
+        size is not supported.
```

Comment:
I specifically wrote `When stride=1`, because it only results in an output size equal to the input size if there's no striding. Would be nice if you could retain this!